dev-libs/vectorscan/metadata.xml

```diff
@@ -0,0 +1,18 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!DOCTYPE pkgmetadata SYSTEM "https://www.gentoo.org/dtd/metadata.dtd">
+<pkgmetadata>
+	<maintainer type="person" proxied="yes">
+		<email>arkamar@atlas.cz</email>
+		<name>Petr Vaněk</name>
+	</maintainer>
+	<maintainer type="project" proxied="proxy">
+		<email>proxy-maint@gentoo.org</email>
+		<name>Proxy Maintainers</name>
+	</maintainer>
+	<slots>
+		<subslots>libhs soname version</subslots>
+	</slots>
+	<upstream>
+		<remote-id type="github">VectorCamp/vectorscan</remote-id>
+	</upstream>
+</pkgmetadata>
```

Comment:
I would personally like some longdescription about this being a continued fork from hyperscan.